gamemodes/terrortown/gamemode/shared/hud_elements/tttsidebar/pure_skin_sidebar.lua

```diff
@@ -81,7 +81,13 @@ if CLIENT then
 		surface.SetDrawColor(item.hud_color.r, item.hud_color.g, item.hud_color.b, math.Round(factor * 255))
 		surface.DrawRect(pos.x, curY, size.w, size.w)
 
-		util.DrawFilteredTexturedRect(pos.x, curY, size.w, size.w, item.hud, iconAlpha, fontColor)
+
+		local hud_icon = item.hud
+		if not hud_icon.GetTexture then
```

Comment:
missing linebreak in front

Reply:
maybe using `istable(hud_icon)` check too?

Reply:
Same problem here